app/stores/interactive-filters.tsx

```diff
@@ -133,15 +135,69 @@ const interactiveFiltersStoreCreator: StateCreator<State> = (set) => {
 type InteractiveFiltersContextValue = [
   UseBoundStore<StoreApi<State>>["getState"],
   UseBoundStoreWithSelector<StoreApi<State>>,
+  StoreApi<State>,
 ];
 
+type SharedFilters = {
+  type: "timeRange" | "dataFilters";
+  value?: NonNullable<InteractiveFiltersConfig>["timeRange"];
+  iri: string;
+}[];
 const InteractiveFiltersContext = createContext<
   | {
       stores: Record<ChartConfig["key"], InteractiveFiltersContextValue>;
+      sharedFilters: SharedFilters;
     }
   | undefined
 >(undefined);
 
+export const getSharedFilters = (
+  chartConfigs: ChartConfig[]
+): SharedFilters => {
+  const allActiveFilters = chartConfigs.flatMap((c) => {
+    const interactiveFiltersConfig = c.interactiveFiltersConfig;
+    if (!interactiveFiltersConfig) {
+      return [];
+    }
```

Comment:
Is it possible that the `interactiveFiltersConfig` is undefined? I think we have it for every chart type 🤔

Reply:
Tables do not have it.

<img width="843" alt="image" src="https://github.com/visualize-admin/visualization-tool/assets/465582/5396907b-31c5-4c3c-a7db-c19715444afd">